cmd/buildx/main.go

```diff
@@ -28,7 +27,6 @@ import (
 )
 
 func init() {
-	seed.WithTimeAndRand()
```

Comment:
I think like BuildKit we need to keep it for now: https://github.com/moby/buildkit/blob/eb3c9d44366e1dbb1de4768d271686b69dd43201/cmd/buildkitd/main.go#L77-L78

@AkihiroSuda Is there a better migration path for this?

Reply:
I don't think it's required - it only calls `math`s `rand.Seed`, and we only use that in one place, where we now explicitly construct it for the pod chooser in the kubernetes driver.

Reply:
Vendored dependencies might be depending on `rand.Seed` too?

Reply:
Ah good point, I'll keep this in for now then - looks like some of the kubernetes code uses it for doing random backoff, and similar.